doc/guides/device_mgmt/dfu.rst

```diff
@@ -36,14 +36,24 @@ In order to use MCUboot with Zephyr you need to take the following into account:
 
 1. You will need to define the flash partitions required by MCUboot; see
    :ref:`flash_map_api` for details.
-2. Your application's :file:`.conf` file needs to enable the
+2. You will have to specify your flash parition as the chosen code partition,
+   see below
+3. Your application's :file:`.conf` file needs to enable the
    :kconfig:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option in order for Zephyr to
    be built in an MCUboot-compatible manner
-3. You need to build and flash MCUboot itself on your device
-4. You might need to take precautions to avoid mass erasing the flash and also
+4. You need to build and flash MCUboot itself on your device
+5. You might need to take precautions to avoid mass erasing the flash and also
    to flash the Zephyr application image at the correct offset (right after the
    bootloader)
 
+.. code-block:: devicetree
```

Comment:
Can this block be put just after 2, to make it more clear? If it renders nicely, I think it's better.

Reply:
Renders better than I initially expected. Done.